packages/components/src/Page/Page.tsx

```diff
@@ -107,6 +114,15 @@ export function Page({
     <div className={pageStyles}>
       <Content>
         <Content>
+          {breadcrumb && (
+            <Button
+              icon="backArrow"
+              url="../"
+              label={breadcrumb}
+              type="tertiary"
+              size="small"
+            ></Button>
+          )}
```

Comment:
if multi-level breadcrumbs were to be a possibility, we could make it so that the breadcrumb type is an array, if the length is 1, use the button style and then some other type of style for multiple?

```typescript
/**
   * Label for the "back" button when Page is nested.
   * The breadcrumb button follows a relative URL to jump up one level.
   */
  readonly breadcrumbs?: Breadcrumb[];
```

```typescript
interface Breadcrumb {
  label: string;
  /* Default = "../" */
  url?: string;
  onClick?(): void;
}
```